### ISSUE TITLE
Make real-time chat more accessible

### ISSUE THREAD

**Issue by hollsk** [pa11y/org#9]
The Slack desktop app has some accessibility problems (the iOS app seems to be pretty good - see https://www.marcozehe.de/2016/01/16/status-of-the-accessibility-of-slack/ and https://blog.echidna.ca/article/hardly-slacker-slack-and-accessibility ). 

We need to look at our options to make sure we're not excluding people. Slack has an IRC bridge that we can enable for people who prefer to use an IRC client: https://get.slack.help/hc/en-us/articles/201727913-Connect-to-Slack-over-IRC-and-XMPP - we need to test this. 

We could also consider switching from Slack to Gitter, which is free, connected to Github accounts, and with better accessibility on the desktop. 

There's a service called sameroom.io that would allow us to bridge between Slack and Gitter (like a11y.slack.com and web-a11y.slack.com), but it costs ~£500 a year :(

**Comment by hollsk** [comment]
I came across this HN thread this morning: https://news.ycombinator.com/item?id=11013136

The article it references isn't something I necessarily agree with (it says "move to IRC"), but the conversation it generates is quite interesting. There's a dev in the top comments who helps develop this alternative: https://matrix.org/ (it has bridges for slack and IRC). It's like a decentralised system of chat nodes, if you will. The FAQ is pretty intriguing. 

IMO the more bridges for a particular service the better, because of the _choice_ to users.

**Comment by rowanmanning** [comment]
Cool, Matrix looks interesting and it'd be nice to bridge. If Gitter _is_ accessible enough then I wouldn't be that adverse to dropping Slack entirely for Pa11y.

**Comment by josebolos** [comment]
Closing this for now as we've been using Slack for too long, and I think that it's fair to say that if we decide to get rid of slack it would be replaced by something that can be integrated with our repos like GitHub discussions.
